enclave/src/rpc/mocks/dummy_builder.rs

```diff
@@ -64,8 +72,16 @@ pub fn sign_trusted_call(
     trusted_call: TrustedCall,
     signer: ed25519_core::Pair,
 ) -> TrustedCallSigned {
-    let mr_enclave = [2u8; 32];
-    let shard_identifier = H256::from(mr_enclave);
+    let mr_enclave = [0u8; 32];
+    let shard_identifier = ShardIdentifier::default();
 
-    trusted_call.sign(&KeyPair::Ed25519(signer), 0, &mr_enclave, &shard_identifier)
+    trusted_call.sign(
+        &KeyPair::Ed25519(signer.clone()),
+        lock_storage_and_get_nonce(signer.public().into())
```

Comment:
that's not a dummy builder. A dummy builder should be hardcoded so you know exactly what you can expect within the unit test

Reply:
The dummy builder for signing now has a nonce argument.